### ISSUE TITLE
Duplicate scripts?

### ISSUE THREAD

**Issue by karllark** [BEAST-Fitting/beast#381]
Has split_calalog_using_map.py superceded subdivida_obscat_by_source_density?

**Comment by drvdputt** [comment]
I took a quick look and there is a small conceptual difference.

`split_catalog_using_map` only looks at the position of each source, with respect to a predefined map. Each source is assigned a bin number based on the source density listed by the map tile it belongs to. Whats actually being binned in the source density space is the tiles, not the sources themselves.

`subdivide_obscat_by_source_density` uses the density value that was added to the individual entries of the catalog. This is a more fine grained approach (unless the source density that was added to the catalog was derived from the mapped value anyway). This probably works fine for source density, as it is easy to estimate (only the positions of the other sources are needed).

The method that I developed to measure the background flux, does not always provide a reliable value for individual sources, so I thought it would be better to use the median of the sources in the tile. I do store both values in the catalog though (`indiv_bg` and `bin_median_bg`); maybe the fine grained approach could be useful in some cases.

**Note** A bigger difference is that `split_catalog_using_map` can also be used to split the AST catalog into parts, without having to assign density values to each entry in this catalog (so, simply using position).
